lib/config/validation.ts

```diff
@@ -1088,3 +1099,26 @@ function isFalseGlobal(optionName: string, parentPath?: string): boolean {
 
   return false;
 }
+
+function validateExperimentalFlag(
+  flagName: string,
+  warnings: ValidationMessage[],
+  currentPath: string | undefined,
+): void {
+  const allowedExperimentalFlags = new Set([
+    'disableDockerHubTags',
+    'execGpidHandle',
+    'noMavenPomCheck',
+    'nugetDownloadNupkgs',
+    'repoCacheForceLocal',
+    'yarnProxy',
+    'useOpenpgp',
+  ]);
```

Comment:
move to module scope and export, so we can add a docs test, so that all flags are documented